src/SFA.DAS.ApprenticeCommitments.Api/Controllers/ApprenticeController.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Threading.Tasks;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.ApprenticeCommitments.Application.Commands.CreateRegistrationCommand;
+
+namespace SFA.DAS.ApprenticeCommitments.Api.Controllers
+{
+    [ApiController]
+    public class ApprenticesController : ControllerBase
+    {
+        private readonly IMediator _mediator;
+
+        public ApprenticesController(IMediator mediator)
+        {
+            _mediator = mediator;
+        }
+
+        [HttpPost("apprentices/{id}/email")]
```

Comment:
It would be nice if this was a put or patch but the outer api I think only supports posts fully. Also I'm perfectly happy with limiting things to POST/GET (as I think they are the only real 2 verbs in http)